kernel/kernel-api/src/main/java/io/delta/kernel/internal/SnapshotImpl.java

```diff
@@ -169,16 +168,4 @@ public CreateCheckpointIterator getCreateCheckpointIterator(Engine engine) {
   public Optional<Long> getLatestTransactionVersion(Engine engine, String applicationId) {
     return logReplay.getLatestTransactionIdentifier(engine, applicationId);
   }
-
-  /**
-   * Returns the commit coordinator client handler based on the table metadata in this snapshot.
-   *
-   * @param engine the engine to use for IO operations
-   * @return the commit coordinator client handler for this snapshot or empty if the metadata is not
-   *     configured to use the commit coordinator.
-   */
-  public Optional<TableCommitCoordinatorClientHandler> getTableCommitCoordinatorClientHandlerOpt(
```

Comment:
Can you delete `TableCommitCoordinatorClientHandler`? Or is that part of the planned followup PR?

Reply:
Will be in followup